### DISCUSSION TITLE
Forced to enter TOTP

### DISCUSSION THREAD

**Discussion by nextcloudfan** [the-djmaze/snappymail#1314] [General]
I activated SnappyMail in NC. On the Admin Page i should enter current PW, and ne PW.
And a 2FA Key... But where do I get the QR Code or something else to generate the 2FA??

**Comment by pepelton** [comment]
I am not in NC, so not sure if this applies, but at least in my installation the TOTP field is not required even though it is shown. So for me just the username + pw was necessary and I could leave the TOTP field empty. But this was several versions ago, so this could have changed, but at least worth a try.
